### DISCUSSION TITLE
从一个 NullPointerException 探究 Java 的自动装箱拆箱机制 — 码志

### DISCUSSION THREAD

**Discussion by mzlogin** [mzlogin/blog-comments#226] [Announcements]
https://mazhuang.org/2017/08/20/java-auto-boxing-unboxing/

**Comment by seecsea** [comment]
怎么我在右上角的搜索框里搜索“自动”，只能搜到“使用 gVim 的 Python 自动补全时遇到的问题”，而没搜索到这篇文档呢？

**Comment by mzlogin** [comment]
站内搜索对中文的支持不太行，参见 https://github.com/mzlogin/mzlogin.github.io/issues/32

**Comment by shixinzhang** [comment]
谢谢您的分享，文章很棒！请问可以转载到公众号《安卓进化论》吗？会标明原文链接和地址。

**Comment by mzlogin** [comment]
> @shixinzhang
> 谢谢您的分享，文章很棒！请问可以转载到公众号《安卓进化论》吗？会标明原文链接和地址。

可以的少年~

**Comment by mzlogin** [comment]
> @seecsea
> 怎么我在右上角的搜索框里搜索“自动”，只能搜到“使用 gVim 的 Python 自动补全时遇到的问题”，而没搜索到这篇文档呢？

我刚刚修改了站内文章标题的搜索能力，现在可以搜索到啦~
